### DISCUSSION TITLE
在 MainLayout 中 设置的 ErrorLogger ，UseTabSet="true" 时，异常正常被 OnErrorHandleAsync 捕获处理了 ，但是 SwalService 没有正常弹框

### DISCUSSION THREAD

**Discussion by yc-2503** [dotnetcore/BootstrapBlazor#4573] [Q&A]
`    <ErrorLogger OnErrorHandleAsync="OnErrorHandleAsync">

        <Layout ShowGotoTop="true" ShowFooter="false" SideWidth="0"
                        IsPage="true" IsFullSide="@IsFullSide" IsFixedHeader="@IsFixedHeader" 
                class="@Theme" ShowCollapseBar="true" Menus="@Menus" TabDefaultUrl="/" UseTabSet="true"
                AdditionalAssemblies="new[] { GetType().Assembly,typeof(PMSM.Client.WebApp.Client._Imports).Assembly }">
`

**Comment by ArgoZhang** [comment]
@yc-2503 请提交复现工程，不然无法帮忙解决问题，`ErrorLogger` 再改版，重写了。不知道新版有没有问题。但是还没有发布包
